api/OEP_API_tutorial_part1.ipynb

```diff
@@ -447,7 +455,7 @@
    "metadata": {},
    "source": [
     "<div class=\"alert alert-block alert-info\">\n",
-    "**201** succesfully inserted data!\n",
+    "<b>201</b> succesfully inserted data!\n",
```

Comment:
You could make sure that the format "Response[XXX]" is applied for all 4 files in a subsequent PR